docs/apple-silicon.md

```diff
@@ -0,0 +1,176 @@
+## Notes: CIEL on Apple Silicon
+
+We need to adapt requirements to our environment, and since Apple Silicon is
+different from other Unix-like OSes, also in terms of where libraries
+and their header files are located, i.e. installed via `homebrew`, we
+need to perform a few additional steps to have CIEL working as it should.
+
+First and foremost, make sure that you are running up to date versions of
+Common Lisp implementation (only tested with SBCL), and `asdf`. Also
+make sure to have added Ultralisp to Quicklisp dists.
+
+### SBCL (homebrew)
+
+```example
+$ brew install sbcl
+$ curl -o /tmp/quicklisp.lisp https://beta.quicklisp.org/quicklisp.lisp
+$ sbcl --load /tmp/quicklisp.lisp
+```
+
+```commonlisp
+(quicklisp-quickstart:install)
+(sb-ext:exit)
+```
+
+```example
+$ rm /tmp/quicklisp.lisp
+```
+
+### SBCL via Roswell (homebrew)
+
+```example
+$ brew update && brew install roswell
+$ ros # installs sbcl by default
+$ ros run # verify by starting REPL
+```
+
+### ASDF
+```example
+$ mkdir -p ~/common-lisp && cd ~/common-lisp
+$ git clone https://gitlab.common-lisp.net/asdf/asdf.git
+```
+
+### Update Quicklisp
+
+Needed if SBCL is installed via Roswell:
+
+```commonlisp
+(ql:update-dist "quicklisp")
+```
+
+### Ultralisp
+
+```commonlisp
+(ql-dist:install-dist "http://dist.ultralisp.org/"
+                          :prompt nil)
+(sb-ext:exit)
+```
+
+### Packages and Dependencies
+
+#### Install dependencies via homebrew
+```example
+$ brew update && brew install \
+       zstd \
+       gcc \
+       sdl2 \
```

Comment:
I'm surprised. Why sdl2?

Reply:
To be honest, I am not sure, I can try without it again, but need some time - incapacitated at the moment. The reason I added it, was I encountered some errors related to SDL, but it might have been dated install via homebrew or me being sort of new to the CL ecosystem, residue from brute force testing. . I will try without it. :)

Reply:
sdl2 is listed in the dependencies for nodgui? https://www.autistici.org/interzona/nodgui.html